bridge/bridge/deploy/sagemaker.py

```diff
@@ -594,18 +595,33 @@ def _create_sagemaker_model(self, version: ModelVersion):
                 Tags=[],
             )
         except ClientError as e:
-            logging.error(e)
-            raise e
+            # If this error was caused by the model already existing
+            # then ignore
+            error_msg = e.response["Error"]["Message"]
```

Comment:
`e.response["Error"]["Code"]` seems more appropriate here

Reply:
Hmm, the code is `ValidationException` which is shared by a number of different messages. I only want to ignore an error related to the model existing etc not for example, an invalid name or something like that

Reply:
I see... My though was that if the `code` is distinctive enough, it's a better attribute to rely upon (the message can be localized, change over time etc). But if it's just `ValidationException`, you have no choice...